### PR TITLE
feat(pagination) add base pagination

### PR DESCRIPTION
**Pull request by dope**

### CODE REVIEW

File: src/components/Pagination/index.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+          <path
+            fill="#FFBABA"
+            d="M27.8 12.7c.2.6-.5 1.2-1 .9-2-1.2-2.5-4-2.3-5.6 0-.7.7-.7 1-.1.7 1.2 1.7 3.1 2.3 4.8zM22 18.4c-.3.2-.8-.1-.8-.6 0-1 .2-2.1 1-3.3.4-.4 1-.2 1.2.3.4 1.6.4 2.7-1.3 3.6zM29 16.5c.2.6-.6 1-1 .5-1-.8-1-1.7-.6-2.2.2-.2.4-.3.6-.2.5.3 1 1 1 1.9z"
+          />
+        </g>
+      ) : (
+        false
+      )}
+
+      {/* Third Item */}
```
Comment (glemmaPaul) [inline src/components/Pagination/index.js]: dats ugly

**Review by glemmaPaul — APPROVED** [review]

File: src/components/Pagination/index.js (hunk 1 above)
Comment (dope) [reply src/components/Pagination/index.js]: do it better!

Comment (glemmaPaul) [reply src/components/Pagination/index.js]: u did ❤️

File: src/components/Pagination/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import StepThree from "./StepThree";
+
+const Pagination = props => {
+  const step = props.step;
+
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="51" height="22" fill="none">
+      <Background />
+
+      {step > 0 && <StepOne />}
```
Comment (glemmaPaul) [inline src/components/Pagination/index.js:15]: u smart man `&&`

File: src/components/GratefulForm/GratefulInputItems.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
+import Pagination from "../Pagination";
 
 const GratefulInputItems = (props) => {
   const { index, onChange, onKeyPress, item } = props
   const active = props.active || 0
       return <li
           hidden={(index !== active)}
-          className="list-item list-item--input mt- mb-">
+          className="list-item list-item--input mt- mb- por">
```
Comment (glemmaPaul) [inline src/components/GratefulForm/GratefulInputItems.js:9]: `por knor wor lol for sure-item`

**Review by glemmaPaul — APPROVED** [review]
Looking goowd

**Pull request merged**